src/viam/examples/project/pkg-config/Makefile

```diff
@@ -1,19 +1,27 @@
 CXX = c++
 
-CXXFLAGS = -std=c++14
+# Technically the SDK supports building with c++14, but note that more recent
+# versions of abseil will force c++17, so this may be needed to make things
+# work out of the box.
+CXX_STANDARD ?= c++17
+
+CXXFLAGS += -std=$(CXX_STANDARD)
 CXXFLAGS += `pkg-config --cflags viam-cpp-sdk-libviamsdk`
-LINKFLAGS = `pkg-config --libs viam-cpp-sdk-libviamsdk`
+LDFLAGS += `pkg-config --libs viam-cpp-sdk-libviamsdk`
 
 all: example_dial example_module
 
 example_dial.o: ../../dial/example_dial.cpp
 	$(CXX) $(CXXFLAGS) -c -o $@ $^
 
 example_dial: example_dial.o
-	$(CXX) $(LINKFLAGS) -o $@ $^
+	$(CXX) $(LDFLAGS) -o $@ $^
```

Comment:
It is a good habit to reiterate `CXXFLAGS` on the link line.

Reply:
Ah good to know, will do